src/core/namespace.c

```diff
@@ -76,12 +77,13 @@ typedef struct MountEntry {
         LIST_HEAD(MountOptions, image_options);
 } MountEntry;
 
-/* If MountAPIVFS= is used, let's mount /sys and /proc into the it, but only as a fallback if the user hasn't mounted
+/* If MountAPIVFS= is used, let's mount /sys, /proc, /dev and /run into the it, but only as a fallback if the user hasn't mounted
  * something there already. These mounts are hence overridden by any other explicitly configured mounts. */
 static const MountEntry apivfs_table[] = {
         { "/proc",               PROCFS,       false },
         { "/dev",                BIND_DEV,     false },
         { "/sys",                SYSFS,        false },
+        { "/run",                RUN,          false },
```

Comment:
probably should initialize `.options_const` and use TMPFS_LIMITS_RUN for it. See what `protect_home_tmpfs_table[]` does, as well as what mount-setup.c does for /run.

Reply:
ok, added ```.options_const = "mode=755" TMPFS_LIMITS_RUN, .flags = MS_NOSUID|MS_NODEV|MS_STRICTATIME``` as mount-setup.c does - do I need the smack ifdef too?

Reply:
can't answer that. no experience with smack. leave it out. if smack people care they'll eventually send us a patch.